packages/cli/templates/react/src/App.tsx

```diff
@@ -1,7 +1,5 @@
 import { useFieldPlugin } from './useFieldPlugin'
-import Counter from './components/Counter'
-import ModalToggle from './components/ModalToggle'
-import AssetSelector from './components/AssetSelector'
+import FieldPlugin from './components/FieldPlugin'
 
 function App() {
```

Comment:
```suggestion
import { FunctionComponent } from 'react'

const App: FunctionComponent = () => {
```

So that the function is guaranteed to be a React component. If a developer decides to return something funny, like a string.